twistlock/tests/test_twistlock.py

```diff
@@ -0,0 +1,70 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import mock
+import json
+
+from datadog_checks.twistlock import TwistlockCheck
+
+customtag = "custom:tag"
+
+instance = {
+    'url': 'http://localhost:8081',
+    'tags': [customtag],
+    'ssl_verify': False
+}
+
+METRICS = [
+    'twistlock.registry.cve.details',
+    'twistlock.registry.cve.count',
+    'twistlock.registry.compliance.count',
+    'twistlock.registry.size',
+    'twistlock.registry.layer_count',
+    'twistlock.images.cve.details',
+    'twistlock.images.cve.count',
+    'twistlock.images.compliance.count',
+    'twistlock.images.size',
+    'twistlock.images.layer_count',
+    'twistlock.hosts.cve.details',
+    'twistlock.hosts.cve.count',
+    'twistlock.hosts.compliance.count',
+    'twistlock.containers.compliance.count',
+]
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+class MockResponse:
+    def __init__(self, j):
+        self.text = j
+        self._json = j
+        self.status_code = 200
+
+    def json(self):
+        return json.loads(self._json)
+
+
+def mock_get(url, *args, **kwargs):
+    split_url = url.split('/')
+    path = split_url[-1]
+    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', path)
+    with open(f_name, 'r') as f:
+        text_data = f.read()
+        return MockResponse(text_data)
+
+
+def test_check(aggregator):
+
+    check = TwistlockCheck('twistlock', {}, {})
+
+    with mock.patch('requests.get', side_effect=mock_get, autospec=True):
+        check.check(instance)
+        check.check(instance)
+
+    for metric in METRICS:
+        aggregator.assert_metric(metric)
+        aggregator.assert_metric_has_tag(metric, customtag)
```

Comment:
Are custom tags the only tags to check for? Would it be possible to check for other tages as well?

Reply:
We can check for them yes

Reply:
actually on second thought, I think it'll be too complex to add in the time being. there are so many metrics in this test and they all have so many tags

Reply:
I am happy to go back after and add them but I think it'll take a few days to do this